### PR TITLE
Add netfx configuration to shim project to build netstandard shim

### PR DESCRIPTION
**Pull request by weshaggard**
Contributes to https://github.com/dotnet/standard/issues/274.

Note: This will suffer from the signing issue https://github.com/dotnet/corefx/issues/17494. @crummel do you know when you will be able to look into that issue? 

PTAL @ericstj

### CODE REVIEW

**Comment by weshaggard** [comment]
@dotnet-bot test Innerloop OSX10.12 Release x64 Build and Test please (network issue during tool init).

**Comment by ericstj** [comment]
I would have expected to see a baseline file get added with the missing API.  Where's that?

**Comment by crummel** [comment]
@weshaggard I can take a look early next sprint but if it's not a quick fix I'm not sure I'll have time, it's already looking pretty full for me.

**Comment by weshaggard** [comment]
@crummel please make sure it is brought up in engineering triage as it has the potential to block our preview release coming up.

**Comment by weshaggard** [comment]
>I would have expected to see a baseline file get added with the missing API. Where's that?

Yea I missed those. Added.

File: src/shims/ApiCompatBaseline.netfx.netstandard20.txt [hunk 1]
```diff
@@ -0,0 +1,164 @@
+Compat issues with assembly mscorlib:
```
Comment (ericstj) [inline src/shims/ApiCompatBaseline.netfx.netstandard20.txt]: These look to me like we're not passing in the right set of seeds.  Shouldn't these be handled by the OOB implementations / partial facades?

Comment (weshaggard) [inline src/shims/ApiCompatBaseline.netfx.netstandard20.txt]: The seeds are the RefPath which is correct the issue is that these don't have netfx refs that get bin-placed into the ref\netfx folder. As we work through your support package stuff we can get this baseline down close to zero (there will still be missing members on existing types).

Comment (weshaggard) [inline src/shims/ApiCompatBaseline.netfx.netstandard20.txt]: Also looks like we are going to need to re-enable a number of the partial facade netfx configurations as part of this work.

**Comment by weshaggard** [comment]
I'm going to work on refreshing some of the baselines after the recent standard changes after I merge this.

**Pull request merged**